meta-refkit-industrial/recipes-core/meta/buildtools-tarball.bbappend

```diff
@@ -0,0 +1 @@
+TOOLCHAIN_HOST_TASK_append = " nativesdk-python3-lxml"
```

Comment:
@rojkov let's see if it works. however, this may trigger another failure in the compatibility selftests due to missing override.

Reply:
That's unlikely since `buildtool-tarball.bb` is defined in OE-core.
At least this time the PR has been built successfully.

Reply:
for the signature check it does not matter where the .bb is defined. It works because buildtools-tarball is not part of world (signature checker runs ```bitbake -S none world```)